common/app/conf/switches/JournalismSwitches.scala

```diff
@@ -13,6 +13,16 @@ trait JournalismSwitches {
     exposeClientSide = false
   )
 
+  val RenderInArticleAudioAtomSwitch = Switch(
+    SwitchGroup.Journalism,
+    "render-in-article-audio-atoms",
+    "controls whether in-article audio atoms are displayed",
+    owners = Seq(Owner.withGithub("JustinPinner")),
+    safeState = On,
```

Comment:
should safeState be `Off`?

Reply:
I think `On` by default is ok as we're releasing something we're pretty confident will behave and we know is going to be used very sparingly in the early stages. This is an emergency kill switch in case something unexpected happens, or we choose to temporarily disable the feature while we work on the next iteration etc.